src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -610,6 +619,28 @@ void testBroadCastMessageWithComponent()
 		playerB.assertSaid(PlainTextComponentSerializer.plainText().serialize(component));
 	}
 
+	@Test
+	void testGetPlayerList()
+	{
+		PlayerMock playerA = server.addPlayer();
+		PlayerMock playerB = server.addPlayer();
+
+		assertThat(server.getPlayerList().getOnlinePlayers(), containsInAnyOrder(playerA, playerB));
+		assertInstanceOf(MockPlayerList.class, server.getPlayerList());
```

Comment:
```suggestion
```
Unnecessary as I don't see what else it could be other than null, which could have a test but not part of this. Try to keep tests to only one thing.